README.md

```diff
@@ -59,13 +59,51 @@ npm run watch
 ```
 
 ## Tests
+
+### JavaScript
+
 The JavaScript tests currently only cover a small portion of data processing utilities. You can run them with [Node]:
 
 ```sh
 npm install --dev
 npm test
 ```
 
+### Jekyll filters
+We have created a set of [custom Jekyll filters](https://jekyllrb.com/docs/plugins/#liquid-filters) that can be used for templating. The filters in _plugins/eiti_*.rb are tested with [rubydoctest](https://github.com/tslocke/rubydoctest).
+
+#### Testing filters
+
+You can run the unit tests as follows:
+
+```sh
+npm test-ruby
```

Comment:
The Ruby standard is pretty much that the default Rake task runs all tests. And Ruby projects should be testable in three steps: Clone, bundle, rake:

```bash
$ git clone [...]
$ bundle install
$ rake
```

All the Ruby tests run and should pass. 

Also, it's a little weird to me that you're running Ruby tests in NPM in some version of Ruby that's been ported to Javascript. (?) This sounds like a recipe for problems down the line, due to language version mismatches and complexity. But I'm new to writing Jekyll filters...